src/test/ui/resolve/levenshtein.stderr

```diff
@@ -0,0 +1,56 @@
+error[E0412]: unresolved type `esize`
+  --> $DIR/levenshtein.rs:15:11
+   |
+15 | fn foo(c: esize) {} // Misspelled primitive type name.
+   |           ^^^^^ did you mean `isize`?
+
+error[E0412]: unresolved type `Baz`
+  --> $DIR/levenshtein.rs:19:10
+   |
+19 | type A = Baz; // Misspelled type name.
+   |          ^^^ did you mean `Bar`?
+
+error[E0412]: unresolved type `Opiton`
+  --> $DIR/levenshtein.rs:20:10
+   |
+20 | type B = Opiton<u8>; // Misspelled type name from the prelude.
+   |          ^^^^^^^^^^ did you mean `Option`?
```

Comment:
>have span to only point to `Opiton`

That would be nice, but path segments and identifiers in them don't have their own spans.

>suggest `Option<u8>`

In fact, type arguments are intentionally dropped from paths in resolve error messages - they are usually irrelevant and mostly bring noise. I tried to include them and it looked worse. I can make a separate branch with this change so you can look and compare.